utils/cautils/acmeutils.go

```diff
@@ -423,7 +430,20 @@ func (af *acmeFlow) GetCertificate() ([]*x509.Certificate, error) {
 		if err != nil {
 			return nil, err
 		}
-
+		hasSubject := false
+		for _, n := range idents {
+			if n.Value == af.subject {
+				hasSubject = true
+			}
+		}
+		subjectIsNotAnIP := net.ParseIP(af.subject) == nil
+		if !hasSubject && subjectIsNotAnIP {
```

Comment:
I think we should also add `&& len(ips) == 0`, if not I think you cannot create a certificate with 
```
CN=localhost
IP:[127.0.0.1]
DNS: []
```
And that only if ACME allows you to create certs without a DNS name.